app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -1,8 +1,17 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
-import { generateMainImage } from '../actions';
 
-// const ImageManagement = ({ onExit }) => (
+const TextField = ({ label, id, value, onKeyUp }) => {
+  return (
+    <div>
+      <label htmlFor={id}>
+        {label}
+        <input type="text" value={value} name={id} onKeyUp={onKeyUp} id={id} />
+      </label>
```

Comment:
This is why the UI is broken. You can either move the input outside the label or fix the CSS so that it looks like before.

I think the first option is the easiest!

```suggestion
      <label htmlFor={id}>
        {label}
      </label>
      <input type="text" value={value} name={id} onKeyUp={onKeyUp} id={id} />
```

Reply:
Oh thanks for suggestions. Changes Updated @rhymes